deploy_pulp3/pulp3/tasks/main.yml

```diff
@@ -0,0 +1,102 @@
+---
+- name: Create user {{ pulp_user }}
+  user:
+    name: "{{ pulp_user }}"
+    comment: "Pulp user"
+
+- name: Get user's info
+  getent:
+    database: passwd
+    key: "{{ pulp_user }}"
+
+- name: Set variable referencing user's home directory
+  set_fact:
+    pulp_user_home: "{{ getent_passwd[pulp_user][4] }}"
+
+- name: Make user's home directory world-readable
+  file:
+    path: "{{ pulp_user_home }}"
+    state: directory
+    mode: 0755
+
+- name: Set variable referencing pulpvenv dir
+  set_fact:
+    pulp_venv: "{{ pulp_user_home }}/pulpvenv"
+
+
+- name: Create ~/.bashrc.d/
+  file:
+    path: "{{ pulp_user_home }}/.bashrc.d/"
+    state: directory
+  become_user: "{{ pulp_user }}"
+
+- name: Create directory /var/lib/pulp
+  file:
+    path: /var/lib/pulp
+    state: directory
+    owner: "{{ pulp_user }}"
+    group: "{{ pulp_user }}"
+    mode: 0775
+
+- name: Install pulpcore
+  pip:
+    name: pulpcore
+    state: latest
```

Comment:
Shouldn't this be "present" instead of "latest?" This role is designed for use by end users, and if an end user has a working Pulp installation, then shouldn't we leave it be instead of fiddling with packages under their feet?